tests/test_browsing_context.py

```diff
@@ -871,6 +871,56 @@ async def test_browsingContext_reload_waitComplete(websocket, context_id,
 
 
 @pytest.mark.asyncio
-@pytest.mark.skip(reason="TODO: Not Implemented")
-async def test_browsingContext_ignoreCache():
-    pass
+@pytest.mark.parametrize("ignoreCache", [True, False])
+async def test_browsingContext_ignoreCache(websocket, context_id, ignoreCache):
+    url = "https://example.com/"
+
+    await subscribe(websocket, [
+        "network.beforeRequestSent",
+        "network.responseCompleted",
+    ])
+
+    await goto_url(websocket, context_id, url)
+
+    id = await send_JSON_command(
+        websocket, {
+            "method": "browsingContext.reload",
+            "params": {
+                "context": context_id,
+                "ignoreCache": ignoreCache,
+                "wait": "complete",
+            }
+        })
+
+    response = await read_JSON_message(websocket)
+    assert response == {
+        "method": "network.beforeRequestSent",
+        "params": {
+            "context": context_id,
+            "initiator": ANY_DICT,
+            "navigation": ANY_STR,
+            "redirectCount": 0,
+            "request": ANY_DICT,
+            "timestamp": ANY_TIMESTAMP,
+        },
+    }
+
+    response = await read_JSON_message(websocket)
+    assert response == {
+        "method": "network.responseCompleted",
+        "params": {
+            "context": context_id,
+            "navigation": ANY_STR,
+            "redirectCount": 0,
+            "request": ANY_DICT,
+            "response": AnyExtending({"fromCache": False}),
```

Comment:
One would expect the correct value here is `not ignoreCache`, however it fails.

Reply:
Upon discussing this in internal chat, it doesn't seem to be possible to detect this condition from `network.responseCompleted`.